Source/Streamstone.Tests/Scenarios/Tracking_entity_changes.cs

```diff
@@ -577,15 +586,21 @@ yield return
             }
         }
 
-        void InsertTestEntity(TestEntity entity)
+        void InsertTestEntity(ITableEntity entity)
```

Comment:
Thank you for making the assert message more meaningful.
Yes, this in memory + `DynamicTableEntity`  doesn't look good at all :)
Does it mean that all of the above are addressed?